pkg/scheduler/framework/types.go

```diff
@@ -229,6 +229,8 @@ type FitError struct {
 const (
 	// NoNodeAvailableMsg is used to format message when no nodes available.
 	NoNodeAvailableMsg = "0/%v nodes are available"
+	// PostFilterMsg is used to format message returned from PostFilterPlugins.
+	PostFilterMsg = "preemption status: %v"
```

Comment:
wait, what if it's a different plugin?
Maybe this string can be added in a different location that is specific to the default preemption plugin?

Reply:
postfilter status？

Reply:
Maybe leave it up to the plugin to set a proper message.
I don't think PostFilter is something the end users need to be aware of.

Reply:
Make sense
Done Thanks